### ISSUE TITLE
Improvement: Generating Views, Controllers in ts-app like in generator-ui5-project

### ISSUE THREAD

**Issue by trashcoder** [ui5-community/generator-ui5-ts-app#18]
Generating Views, Controllers etc. like in generator-ui5-project would be a big improvement for generator-ui5-ts-app.

**Comment by petermuessig** [comment]
Yes, indeed. I even think further whether we can't harmonize the different generators. Right now, the project generator is the most advanced providing many sub generators and all others (ts-app, app, library, ...) are very simple self-contained ones.

**Comment by IObert** [comment]
I just wanted to open a issue for exactly this issue :) I'm glad the conversation has already started and, hopefully, I can relive it a bit.
A similar question has already been asked in the issue about 18 month ago ui5-community/generator-ui5-project#17

I noticed ui5-community/generator-ui5-project#58 already brings some TS-support to the default generator. But it is tightly bound to the "SAP Fiori elements flexible programming model"-option. @tobiasqueck Do you know if `@sap-ux/fe-fpm-writer`, `@sap-ux/fiori-freestyle-writer`, or `@sap-ux/ui5-application-writer` provide the capability to generate a "plain TS project"?

**Comment by tobiasqueck** [comment]
@IObert yes, `@sap-ux/ui5-application-writer` allows generating a plain UI5 TS project. The corresponding test cases using the required flag are https://github.com/SAP/open-ux-tools/blob/main/packages/ui5-application-writer/test/options.test.ts#L86-L131
